src/page-modules/contact/travel-guarantee/form-selector.tsx

```diff
@@ -0,0 +1,113 @@
+import { PageText, TranslatedString, useTranslation } from '@atb/translations';
+import { SectionCard } from '../components/section-card';
+import { Input } from '../components/input';
+import style from '../contact.module.css';
+import { RadioInput } from '../components/input/radio';
+import { Typo } from '@atb/components/typography';
+import { Checkbox } from '../components/input/checkbox';
+
+type FormSelectorProps = {
+  state: any;
+  send: any;
+};
+
+export const FormSelector = ({ state, send }: FormSelectorProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <div>
+      {!state.context.isIntialAgreementChecked && (
+        <SectionCard title={PageText.Contact.travelGuarantee.title}>
+          <Typo.p textType="body__primary">
+            {t(
+              PageText.Contact.travelGuarantee.layoutAgreement
+                .delayedRefundText,
+            )}
+          </Typo.p>
+
+          <Typo.h3 textType="heading__component">
+            {t(
+              PageText.Contact.travelGuarantee.layoutAgreement.ticketRefundText,
+            )}
+          </Typo.h3>
+
+          <h4>
+            {t(
+              PageText.Contact.travelGuarantee.layoutAgreement
+                .travelGuaranteeExceptions.label,
+            )}
+          </h4>
+          <Typo.p textType="body__primary">
+            {t(
+              PageText.Contact.travelGuarantee.layoutAgreement
+                .travelGuaranteeExceptions.minimumTimeToNextDeparture,
+            )}
+          </Typo.p>
+
+          <Typo.p textType="body__primary">
+            {t(
+              PageText.Contact.travelGuarantee.layoutAgreement
+                .travelGuaranteeExceptions.externalFactors,
+            )}
+          </Typo.p>
+          <ul className={style.rules__list}>
+            {PageText.Contact.travelGuarantee.layoutAgreement.travelGuaranteeExceptions.examples.map(
+              (example: TranslatedString, index: number) => (
+                <li key={index}>{t(example)}</li>
```

Comment:
Should this include a typography component? I think this would make the text look different from the rest.